bundles/AdminBundle/Resources/public/js/pimcore/startup.js

```diff
@@ -950,18 +950,22 @@ pimcore["intervals"]["translations_admin_missing"] = window.setInterval(function
     var missingTranslations = pimcore.globalmanager.get("translations_admin_missing");
```

Comment:
off topic:

do we currently really need 

translations_admin_missing
and
translations_admin_added

or can we can get rid of one of them ?

Reply:
I'm sure that it's possible to implement an algorithm with just a single array but this would be a separate task.